conans/test/functional/command/profile_test.py

```diff
@@ -69,6 +71,8 @@ def show_test(self):
     def profile_update_and_get_test(self):
         client = TestClient()
         client.run("profile new ./MyProfile --detect")
+        self.assertIn("edit the ./MyProfile profile at", client.out)
+        self.assertIn("profiles/./MyProfile", client.out)
```

Comment:
Is this assert right? Shouldn't it be ``self.assertNotIn("profiles/./MyProfile", client.out)`` ?

Reply:
Yes, you want to see the path of the profiles. 
The only thing is that it is not converted to abs_path, but that might be a bit messy, so I am fine with this PR as-is.